WooCommerce/Classes/ViewRelated/Products/Media/ProductImagesCollectionViewController.swift

```diff
@@ -159,6 +165,15 @@ extension ProductImagesCollectionViewController {
 ///
 extension ProductImagesCollectionViewController: UICollectionViewDragDelegate, UICollectionViewDropDelegate {
     func collectionView(_ collectionView: UICollectionView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
+
+        // If any image is currently being uploaded, display a message to the user and prevent drag & drop operations.
+        guard !productImageStatuses.contains(where: { if case .uploading = $0 { return true }; return false }) else {
```

Comment:
💡 this `if case` syntax stumped me for a little bit (it makes sense but got me thinking "why are this written this way?"). Luckily [this helps me understand](https://stackoverflow.com/a/37888514).